actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -206,6 +207,12 @@ private void updateUser(Request actorMessage) {
     Map<String, Boolean> userBooleanMap = updatedUserFlagsMap(userMap, userDbRecord);
     int userFlagValue = userFlagsToNum(userBooleanMap);
     requestMap.put(JsonKey.FLAGS_VALUE, userFlagValue);
+    if (StringUtils.isNotEmpty((String) userDbRecord.get(JsonKey.MANAGED_BY))
```

Comment:
can you add a comment - to who will this apply? A regular user or managed user. 
A managed user cannot update an email or phone (UI will not allow it).
If it happens in future, then we will accept it and set managedBy = null. This managed user has become an adult and is no more dependent on the regular user.

Reply:
@indrajra the following is suggested in the [ticket](https://project-sunbird.atlassian.net/browse/SH-135) .

- If an email or phone is added to a user that has a non-null ‘managedBy’, do the following. We shall consider these no more as managed users.
- generate a reset password link